scripts/gh_metrics.py

```diff
@@ -39,11 +42,35 @@ def report():
     multiple=True,
     help="The repository name, like robottelo. ",
 )
+team_name_option = click.option(
+    "--team",
+    default=[],
+    multiple=True,
+    help="The github team name slug (URL field form, like quality-engineers)",
+)
+user_name_option = click.option(
+    "--user",
+    default=[],
+    multiple=True,
+    help="The github login name (URL field form, like mshriver)",
+)
 pr_count_option = click.option(
     "--pr-count",
     default=50,
     help="Number of PRs to include in metrics counts, will start from most recently created",
 )
+num_weeks_option = click.option(
+    "--num-weeks",
```

Comment:
I see we can go by week-wise... you ignore my last comment there ..

Reply:
that will be a great addition , 
another side I was thinking to have date wise option i.e. `--from-date  xx-xx-xx  --to-date xx-xx-xx` will give a more flexible way to go .